### DISCUSSION TITLE
Getting entities without relations

### DISCUSSION THREAD

**Discussion by thanisch** [prisma/prisma#8738] [Q&A] [answered]
This is my schema:

```prisma
model User {
  id        String    @id @default(uuid())
  email     String    @unique @db.VarChar(255)
  createdAt DateTime  @default(now())
  roles     Role[]
}

model Role {
  id                 String           @id @default(uuid())
  createdAt          DateTime         @default(now())
  updatedAt          DateTime?
  title              String
}
```

My question now is:

Is there a way to get all the users, which do not have a role (--> `user.role = []`) with the Prisma Client??

**Comment by ryands17** [comment, marked as answer]
@thanisch 👋 

Yes. Have a look at this section in the docs: https://www.prisma.io/docs/support/help-articles/finding-entities-based-on-relation#solution

**Reply by thanisch** [reply]
@ryands17, thank you so much, went through the whole docs but help section. 😄

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
